tests/system/lab/remove

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+time sudo apt remove -y foglamp foglamp-gui # better to purge?
+sudo rm -rf /usr/local/foglamp # Should not be needed.
```

Comment:
Its better to remove everything (i.e installation dir) as `FogLAMP` package un-installation does not delete its `data` dir